set.mm

```diff
@@ -529347,23 +529401,28 @@ axioms listed above ( ~ axc11 appears slightly stronger since
 
   $( A general FOL biconditional that generalizes ~ 19.9ht among others.  For
      this and the following theorems, see also ~ 19.35 , ~ 19.21 , ~ 19.23 .
-     (Contributed by BJ, 20-Oct-2019.) $)
+     When ` ph ` is substituted for ` ps ` , both sides expreass a form of
+     nonfreeness.  (Contributed by BJ, 20-Oct-2019.) $)
   bj-biexal1 $p |- ( A. x ( ph -> A. x ps ) <-> ( E. x ph -> A. x ps ) ) $=
     ( wal nfa1 19.23 ) ABCDCBCEF $.
 
-  $( A general FOL biconditional.  (Contributed by BJ, 20-Oct-2019.) $)
+  $( When ` ph ` is substituted for ` ps ` , both sides expreass a form of
+     nonfreeness.  (Contributed by BJ, 20-Oct-2019.) $)
   bj-biexal2 $p |- ( A. x ( E. x ph -> ps ) <-> ( E. x ph -> A. x ps ) ) $=
     ( wex nfe1 19.21 ) ACDBCACEF $.
 
-  $( A general FOL biconditional.  (Contributed by BJ, 20-Oct-2019.) $)
+  $( When ` ph ` is substituted for ` ps ` , both sides expreass a form of
+     nonfreeness.  (Contributed by BJ, 20-Oct-2019.) $)
   bj-biexal3 $p |- ( A. x ( ph -> A. x ps ) <-> A. x ( E. x ph -> ps ) ) $=
     ( wal wi wex bj-biexal1 bj-biexal2 bitr4i ) ABCDZECDACFZJEKBECDABCGABCHI $.
 
-  $( A general FOL biconditional.  (Contributed by BJ, 20-Oct-2019.) $)
+  $( When ` ph ` is substituted for ` ps ` , both sides expreass a form of
```

Comment:
expreass -> express here, and more in the following lines

Reply:
Applied in e896920c9d958feded58b3bfdaa007c5c1184635